doc/digital-receipt/Austria/digital-receipt-austria.md

```diff
@@ -0,0 +1,35 @@
+---
+slug: /poscreators/middleware-doc/digital-receipt/austria
+title: 'Digital Receipt: Austria'
+---
+
+# Compliance AT
+
+Vorschriften an den digitalen Beleg in Österreich:
+
+Ein elektronischer Beleg iSd § 132a Abs. 1 BAO ist ein Beleg, der in einem elektronischen Format ausgestellt und empfangen bzw. unmittelbar für den die Barzahlung Leistenden verfügbar ist. Er kann z.B. mittels E-Mail, als E-Mail-Anhang oder Web-Download, in einem elektronischen Format (z.B. als PDF- oder Textdatei), aber auch in einem strukturierten Dateiformat (z.B. HTML) ausgestellt werden. Der Belege muss unmittelbar im Zusammenhang mit der Barzahlung durch die Registrierkasse erstellt und signiert werden, sowie in der Folge tatsächlich in den Verfügungsbereich des Belegempfängers gelangen. Technisch bedingte Verzögerungen wie z.B. bei der E-Mail-Zustellung oder dem Upload zu einem Server sind unbeachtlich, wenn die elektronischen Belege sofort im DEP sowie zeitnah in einem revisionssicheren Speicher (vgl. Abschnitt 3.2.1.4. Sicherung des DEP) gespeichert und für eine Überprüfung durch ein Organ der Finanzverwaltung verfügbar sind. Ob dies elektronisch (per Email, Ablesen mittels Smartphone, usw.) oder in Papierform erfolgt, bleibt dem Belegaussteller überlassen. Die Übermittlung ist eine Bringschuld des Unternehmers, eine bloße Einräumung der Möglichkeit des Ansehens und Abfotografierens des auf einem Bildschirm angezeigten Beleginhaltes erfüllt nicht die Belegerteilungspflicht. Eine spezielle Form der elektronischen Übertragung ist nicht vorgeschrieben. Ob die Übertragung in den Verfügungsbereich des Belegempfängers durch Aktivität des Leistungserbringers oder den die Bezahlung Leistenden erfolgt, ist für die Erfüllung der Belegerteilungspflicht unerheblich. Diese Art der Belegerteilung ist im elektronischen Aufzeichnungssystem zu dokumentieren.
```

Comment:
An digital receipt within the meaning of Section 132a (1) BAO is a receipt that is issued and received in an digital format or is immediately available to the person making the cash payment. It can be issued, for example by web in an digital format (e.g. as a PDF or text file), but also in a structured file format (e.g. HTML). The receipt must be created and signed by the cash register directly in connection with the cash payment, and then actually reach the disposal area of the receipt recipient. Technical delays such as e.g. uploading to a server are irrelevant if the receipts are stored immediately in the DEP and promptly in an audit-proof memory and for a verification by a financial management body is available. It is up to the issuer of the digital receipt whether this is to be done digitally (by using a smartphone, etc.) or in paper form. The transmission is an obligation of the entrepreneur, a mere granting of the possibility of viewing and photographing the content of the receipt displayed on a screen does not fulfill the obligation to issue receipts. A special form of digital transmission is not required. Whether the transfer to the area of disposal of the recipient of the receipt takes place through the activity of the service provider or the person making the payment is irrelevant for the fulfillment of the obligation to issue the receipt. This type of receipt is to be documented in the electronic recording system.